proposals/0001-dep-process.md

```diff
@@ -0,0 +1,312 @@
+
+Title: **DEP-0001: The Dat Enhancement Proposal Process**
+
+Short Name: `0001-dep-process`
+
+Type: Process
+
+Status: Draft (as of 2018-01-15)
+
+Github PR: [Draft](https://github.com/datprotocol/DEPs/pull/2)
+
+Authors: Dat Protocol Working Group: [Tara](https://github.com/taravancil),
+[Paul](https://github.com/pfrazee), [Mathias](https://github.com/mafintosh), [Karissa](https://github.com/karissa), [Joe](https://github.com/karissa), [Danielle](#),
+[Bryan](https://github.com/bnewbold)
+
+# Summary
+[summary]: #summary
+
+The Dat Enhancement Proposal ("DEP") process is how the Dat open source
+community comes to (distributed) consensus around technical protocol
+enhancements and organizational process.
+
+# Motivation
+[motivation]: #motivation
+
+The Dat protocol is still a living standard. A transparent process is needed
+for community members to understand what changes are in the pipeline and how
+new ideas might come to fruition.
```

Comment:
Preparing to handle protocol version changes/breaks is also a primary motivation =). Should be mentioned upfront.

Reply:
Almost forgot this!